payments_py/ai_query_api.py

```diff
@@ -221,9 +222,9 @@ async def subscribe_task_logs(self, callback: Any, tasks: List[str]):
             if not tasks:
                 raise Exception('No task rooms to join in configuration')
             await self.connect_socket()
-            
-            await self.socket_client.on('_connected', self._on_connected(callback, tasks))
+            self.socket_client.on('_connected', self._on_connected(callback, tasks))
         except Exception as error:
+            print(error)
```

Comment:
The library shouldn't print messages, just throw exceptions or return objects with execution results

Reply:
Right, forgot to remove. Fixed